sources/packages/backend/libs/integrations/src/esdc-integration/e-cert-integration/e-cert-file-handler.ts

```diff
@@ -88,20 +89,31 @@ export abstract class ECertFileHandler extends ESDCFileHandler {
     log.info(
       `Retrieving ${offeringIntensity} disbursements to generate the e-Cert file...`,
     );
-
-    const sequenceGroup = `${sequenceGroupPrefix}_${getISODateOnlyString(
+    const fileNameSequenceGroup = `${sequenceGroupPrefix}_${getISODateOnlyString(
       new Date(),
     )}`;
     let uploadResult: ECertUploadResult;
+    let fileInfo: CreateRequestFileNameResult;
+    // Consume the next sequence number for the e-Cert filename
+    // and execute the creation of the request filename.
+    await this.sequenceService.consumeNextSequence(
+      fileNameSequenceGroup,
+      async (nextSequenceNumber: number) => {
+        // Create the request filename with the file path for the e-Cert File.
+        fileInfo = this.createRequestFileName(fileCode, nextSequenceNumber);
+      },
```

Comment:
Not a blocker. Inside the callback, we can just set the `fileNameSequenceNumer`.